src/javascript/binary/reality-check.js

```diff
@@ -165,8 +164,8 @@ RealityCheck = (function ($) {
             $('#reality-check').remove();
         });
 
-        $('#reality-check .blogout').on('click', function () {
-            window.location.href = logout_url;
+        $('#reality-check #btn_logout').unbind('click').click(function(){
+            BinarySocket.send({"logout": "1"});
```

Comment:
will this redirect to login page after logging out? if this is not the case then client will still see logind and logout button in header